/triton/controls/thrust_allocator:
  ros__parameters:
      # Thrusters will follow the following naming convention: 

#                            __
#                           |  |Thruster 6 ___
#                       ____|__|__________|___| Thruster 2 (angled)
#                       |                 |
#                  _____|       +y        |
#              ___|╭───╮       |          |_
# Thruster 1  |___|│ 3 │       |___+x     |_) AUV front 
#                 |╰───╯       +z         |
#                  ‾‾‾‾‾|  (pointing up)  |
#                       |_________________|___
#                           |  |          |___| Thruster 4 (angled)
#                           |__|Thruster 5

    num_thrusters: 6
    bits_per_thruster: 5
    max_fwd: 3.71 # kgf
    max_rev: 2.92 # kgf
    # t3:
    #   contrib:
    #     z: 1.0
    #   lx: -0.219431
    #   ly: -0.055
    #   lz: 0.09
    t5:
      contrib:
        x: -1.0
      lx: -0.075
      ly: -0.3984
      lz: 0.0
    t6:
      contrib:
        x: -1.0
      lx: -0.075
      ly: 0.3984
      lz: 0.0
